packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart

```diff
@@ -41,14 +40,14 @@ import 'package:intl/intl.dart' as intl;
 /// Please make sure to update your pubspec.yaml to include the following
 /// packages:
 ///
-/// ```
+/// ```yaml
 /// dependencies:
 ///   # Internationalization support.
 ///   flutter_localizations:
 ///     sdk: flutter
-///   intl: any # Use the pinned version from flutter_localizations
+///   intl: any # Pinned by flutter_localizations
```

Comment:
Why this change?

Reply:
Oops, might be a random change that should not submit.